pyproject.toml

```
@@ -4,7 +4,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "typedb-jupyter"
-version = "0.3"
+version = "0.4"
```

```suggestion
version = "0.3"
```
This version should only be bumped when released to PyPI.